src/boards/nrf52833/board.h

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
which board is this nrf52833, it is to generic, maybe adding an manufacturer name to distinguish it from other 833 boards.